etc/agent.properties

```diff
@@ -78,14 +78,23 @@ log-entry.default = 0000-01-01T00:00:00.000+0000\
 #### Presto Information #############################################
 
 # These configuration properties tell Presto Manager where to look
-# for Presto's files.
-
+# for Presto's files. The paths can be absolute or relative to the
+# Presto-Manager directory.
 # Presto-Manager will overwrite the `node.properties` file present in
 # `defaults.config-dir` with the below values to prevent conflicts.
 
-presto.config-dir     = /etc/presto
-presto.catalog-dir    = /etc/presto/catalog
-presto.data-dir       = /var/lib/presto/presto/data
-presto.plugin-dir     = /usr/lib/presto/lib/plugin
-presto.log-dir        = /var/log/presto
-presto.launcher       = /usr/lib/presto/bin/launcher
+# for RPM installation
+
+presto.catalog-dir      = /etc/presto/catalog
+presto.data-dir         = /var/lib/presto/data
+presto.log-dir          = /var/log/presto
+
+# for TARBALL installation
+# presto.launcher-Properties is optional
+
+# presto.installation-dir = /usr/presto
```

Comment:
but they're misleading because the directories for tar ball install should all be inside the installation directory